src/pytest_flyte/plugin.py

```diff
@@ -81,19 +106,20 @@ def flyte_workflows_source_dir(pytestconfig):
 
 
 @pytest.fixture(scope="session")
-def flyte_workflows_register(docker_compose):
-    docker_compose.execute("exec backend -w /flyteorg/src make register")
+def flyte_workflows_register(request):
+    proto_path = request.config.getoption("--proto-path")
+    subprocess.check_call(f"bin/flytectl register file {proto_path}", shell=True)
```

Comment:
You also need to add support for tar.gz files. Check https://docs.flyte.org/projects/flytectl/en/latest/gen/flytectl_register_files.html